physics/mirror_formulae.py

```diff
@@ -81,10 +81,10 @@ def focal_length(distance_of_object: float, distance_of_image: float) -> float:
 
 def object_distance(focal_length: float, distance_of_image: float) -> float:
     """
-    >>> object_distance(30, 20)
-    -60
-    >>> object_distance(10.5, 11.7)
-    102.375
+    >>> isclose(object_distance(30, 20), -60.0)
+    True
+    >>> isclose(object_distance(10.5, 11.7), 102.375)
+    True
```

Comment:
I do not think we need is_close() here.  Let’s only use it where we need to.

Reply:
When running the doctests locally, there were issues when used without the isclose, should i still remove them or leave it as it is ?
![image](https://github.com/TheAlgorithms/Python/assets/143083087/d5d51a52-3464-452b-a9c8-4414a884b2f3)

![image](https://github.com/TheAlgorithms/Python/assets/143083087/823ef63d-d19d-484f-8d92-91a323e5afa2)

Reply:
Ok.  We need is_close()

Reply:
Is there anything else i add/change ?, or just have to wait till PR is merged ?